src/material/progress-spinner/progress-spinner.ts

```diff
@@ -33,12 +33,6 @@ export type ProgressSpinnerMode = 'determinate' | 'indeterminate';
  */
 const BASE_SIZE = 100;
 
-/**
- * Base reference stroke width of the spinner.
- * @docs-private
- */
-const BASE_STROKE_WIDTH = 10;
```

Comment:
The intention behind using `BASE_STROKE_WIDTH` constant isn't yet apparent to me, given it is not used anywhere else. Maybe it worked for a spinner with a default diameter and stroke width?

Reply:
Decided to keep `BASE_STROKE_WIDTH` for use in calculating the base circle diameter.